### PR TITLE
predict_metab and predict_DO for metab_simple

### PR DESCRIPTION
**Pull request by aappling-usgs**
review and merge, please, @jread-usgs 

check() gives this NOTE:
```* checking R code for possible problems ... NOTE
predict_DO.metab_simple: no visible binding for global variable
  'date.time'
predict_DO.metab_simple: no visible binding for global variable '.'```
which occurs in a dplyr chain. I don't see any actual flaws in that chain - do you, @jread-usgs? I tried shifting the with() block into its own function, but that didn't change the NOTE.

### CODE REVIEW

File: R/metab_simple.R [hunk 1]
```diff
@@ -88,5 +91,92 @@ metab_simple <- function(data, ...) {
...
+#' @family predict_DO
+predict_DO.metab_simple <- function(metab_model) {
+
+  # get the metabolism (GPP, ER) estimates
+  metab_ests <- predict_metab(metab_model)
+  
+  # re-process the input data with the metabolism estimates to predict dissolved
+  # oxygen
+  get_data(metab_model) %>%
+    mutate(date=as.Date(format(date.time, "%Y-%m-%d"))) %>%
```
Comment (jread-usgs) [inline R/metab_simple.R]: to spoof the R checks, you have to do goofy things before you use `date.time` in cases like this. 
I prefer something like this:
```R
date.time = '.dplyr.var'
```

Comment (jread-usgs) [inline R/metab_simple.R]: This is because of the non-standard eval for dplyr

Comment (aappling-usgs) [inline R/metab_simple.R]: That check() NOTE comes from this dplyr chain right here. The note, again:
```* checking R code for possible problems ... NOTE predict_DO.metab_simple: no visible binding for global variable 'date.time' predict_DO.metab_simple: no visible binding for global variable '.'```

Comment (aappling-usgs) [inline R/metab_simple.R]: uhhh...could you explain a little further, please? where do you set date.time to a string, and why?

Comment (jread-usgs) [inline R/metab_simple.R]: I would to it on line 153 maybe? Somewhere above the mutate call. Basically, R thinks you pulled the `date.time` out of nowhere, so it is confused where it comes from. I see there is a different recommendation by hadley here: http://stackoverflow.com/questions/9439256/how-can-i-handle-r-cmd-check-no-visible-binding-for-global-variable-notes-when

**Pull request merged**